### ISSUE TITLE
'undefined' is added in each line of the code

### ISSUE THREAD

**Issue by HeySora** [isagalaev/highlight.js#1456]
In HTML, I included
```html
<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/styles/monokai-sublime.min.css" />
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/highlight.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/languages/lua.min.js"></script>
```

In JS, I'm doing
```js
hljs.configure({useBR: true});
$('div.example').each(function(i, v) {
	hljs.highlightBlock(v);
});
```

But now :
```html
<div class="example lua">self:cmd(\'addx,30;zoom,2\'); -- Does :addx(30), :zoom(2)</div>
```
becomes (I removed the hljs spans here)
```html
<div class="example lua hljs">undefinedself:cmd(\'addx,30;zoom,2\'); -- Does :addx(30), :zoom(2)</div>
```

Did I do something wrong ?

**Comment by HeySora** [comment]
This is only occuring with useBR set to true.

**Comment by isagalaev** [comment]
Looks like this exact bug was fixed recently in #1453. Could you check if it affects you with the current master code?

**Comment by HeySora** [comment]
Yes, it fixed it, thanks !
